### DISCUSSION TITLE
How to get the latest version of Figma UI KIT

### DISCUSSION THREAD

**Discussion by dwiasri500** [primefaces/community#1622] [Figma UI Kit]
Hi all, 
I currently saw there is a stepper here:
<img width="1401" alt="Screenshot 2024-04-08 at 15 33 09" src="https://github.com/primefaces/community/assets/124853649/3fb4581f-5524-491e-95b5-662947a221c1">

I purchased the UI Kit but in my files, I couldn't find the stepper. What should I do to get it?

**Comment by onursenture** [comment]
Stepper, InputOTP, MeterGroup are newly released components. We will be including these in future versions of the Figma UI Kit. We are currently still working on this. Thank you!
